app/(admin)/layout.tsx

```diff
@@ -31,10 +42,88 @@ const RootLayout = ({
 }): JSX.Element => {
   return (
     <html lang="en" className={GeistSans.className}>
-      <body className="dark:dark bg-background pb-8 px-4 text-foreground xl:pb-0">
+      <body className="dark:dark bg-background text-foreground">
         <ErrorBoundary>
           <AuthContextProvider>
-            <main>{children}</main>
+            <main className="grid grid-cols-adminLayout h-screen">
+              <section className="left-column grid grid-rows-adminLayout h-full border-r-2 border-border">
```

Comment:
Why use `grid-rows-adminLayout` twice?

Reply:
It's not being used twice here. The first is grid-_cols_ and the other is grid-_rows_.

I have the page structured so both columns are in one parent container (`main`) which is where the columns are defined. Then in the two children `section`s of `main` I'm defining `grid-rows-adminLayout`.